src/components/Dashboard/CustodianListSm.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div id="custodian-list-sm">
```

Comment:
Consider a v-if here to display nothing if the custodians array is empty. (Or it that already checked in the parent?)

Reply:
I see you already check that in the Dashboard, so all good.